src/Engine/Component/PathfindingComponents/PathfindingComponent.cpp

```diff
@@ -1,12 +1,28 @@
 #include "PathfindingComponent.hpp"
+
+#include "glm/gtx/norm.hpp"
+
 #include "System/PathfindingSystem.hpp"
 #include "Component/SpatialComponents/SpatialComponent.hpp"
 
-void PathfindingComponent::init() {}
+PathfindingComponent::PathfindingComponent(GameObject & player, float ms) :
+    m_spatial(nullptr),
+    m_player(&player),
```

Comment:
What if we want a pathfinding component that is attached to something other than the player?

Reply:
I would still like some clarification on this